environment.yml

```diff
@@ -40,39 +41,39 @@ dependencies:
   - nc-time-axis
   - netCDF4
   - numba
-  - numpy !=1.24.3  # severe masking bug
+  - numpy !=1.24.3,<2.0  # severe masking bug
   - openpyxl
   - packaging
-  - pandas !=2.2.0,!=2.2.1,!=2.2.2 # github.com/ESMValGroup/ESMValCore/pull/2305
```

Comment:
Maybe best to keep this, as ESMValTool is works with ESMValCore v2.11.0, which does not have the workaround that has been implemented in https://github.com/ESMValGroup/ESMValCore/pull/2529

Reply:
agh - icon cmorizer - yeah, hang on, that may still be needed, lemme think for a sec

Reply:
OK repinned including latest 2.2.3 in d1de6875aacf2b53abbb9f414af263719f3f7b72 and added 2.2.3 to setup.py pins in d5676266f44b914f88b2914ac56c317484b5f61e

Reply:
But then we should probably pin <2.2.0. Otherwise this will break once pandas releases a new version.

Reply:
well - how about I exact pin it to the earliest latest version that works for us ie 2.1.4 to avoid some ye olde panda getting in the env and easing a bit Mamba's efforts?

Reply:
OK I fixpinned pandas==2.1.4 and added a note in ecc1d6e57f6d3c0158215581c9345b52a9acd546 and acaca7c93773f54e210af58503cbc907270ca121